src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,135 @@
+package duke;
+
+import java.util.ArrayList;
+
+import duke.task.Task;
+
+/**
+ * Ui is a class that prints messages to the user.
+ */
+public class Ui {
+    /**
+     * Prints a welcome message.
+     */
+    public static void welcomeMessage() {
```

Comment:
Can consider changing to printWelcomeMessage() so as to make it a verb (ie an action)